cli/cmd/statusStep.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright © 2021 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"github.com/spf13/cobra"
+	"bytes"
+	"encoding/json"
+	"strconv"
+	
+)
+
+// statusPipelineCmd represents the statusPipeline command
+var statusStepCmd = &cobra.Command{
+	Use:   "step <step name> <team name> <pipeline name> ",
+	Short: "Fetch the status of a pipeline step",
+	Long: `
+Command to fetch the status of a pipeline step. Specify the name of the step as the argument.
+Use the --team and --pipeline flags to specify the team and the pipeline this step belongs to.
+The step count can optionally be set with the -c flag, otherwise it will be set to 15 by default.
+ 
+Example usage:
+	atlas status step step_name --team team_name --pipeline pipeline_name  (No -c flag specified means count is 15)
```

Comment:
atlas logs get examplePipeline
atlas logs get examplePipeline --step deploy_to_dev

Reply:
Also double check status api in case of http header issue